src/css/main.css

```diff
@@ -11533,19 +11533,13 @@ h1, h2, h3, h4, h5, h6 {
   }
 }
 
-{/* If the width is greater than 992, then don't show the component */}
+{/* If the width is greater than 992, then don't show the component, in Cordova this is overwritten to 741px in MuiTheme.js */}
 @media all and (min-width: 992px) {
   .u-show-mobile-tablet {
     display: none !important;
   }
 }
 
-@media all and (max-width: 991px) {
```

Comment:
Hi @SailingSteve u-show-desktop is still used in many places and unless I'm mistaken, this CSS can't be removed.

Reply:
No, all the u-show-desktop references are gone

Reply:
src/js/components/Navigation/HeaderBar.jsx line 518

Reply:
Ok, fixed that one.